.travis.yml

```diff
@@ -0,0 +1,63 @@
+sudo: false
+dist: trusty
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+matrix:
+  include:
```

Comment:
We probably only need to test `latest` and `trunk`,

Reply:
OK

Reply:
Sure, https://github.com/getblocklab/block-lab/pull/276/commits/f41885acabb849c45be645209a353c92c104492b removes WordPres 4.5 from the build.